static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -68,348 +94,434 @@ function createSeriesAndCount(stats: EventsStats) {
-            disabled
-            label={t('Events')}
-          />
-          <GraphButton
-            isActive={visibleSeries === EventGraphSeries.USER}
-            disabled
-            label={t('Users')}
+        {showSummary ? (
+          <SummaryContainer>
+            <GraphButton
+              onClick={() =>
+                visibleSeries === EventGraphSeries.USER &&
+                setVisibleSeries(EventGraphSeries.EVENT)
+              }
+              isActive={visibleSeries === EventGraphSeries.EVENT}
+              disabled={visibleSeries === EventGraphSeries.EVENT}
+              label={tn('Event', 'Events', eventCount)}
+              count={String(eventCount)}
+            />
+            <GraphButton
+              onClick={() =>
+                visibleSeries === EventGraphSeries.EVENT &&
+                setVisibleSeries(EventGraphSeries.USER)
+              }
+              isActive={visibleSeries === EventGraphSeries.USER}
+              disabled={visibleSeries === EventGraphSeries.USER}
+              label={tn('User', 'Users', userCount)}
+              count={String(userCount)}
+            />
+          </SummaryContainer>
+        ) : (
+          <div />
+        )}
+        <ChartContainer role="figure">
+          <BarChart
+            {...releaseBubbleEventHandlers}
+            ref={handleConnectRef}
+            height={100}
+            series={series}
+            additionalSeries={releaseBubbleSeries ? [releaseBubbleSeries] : []}
+            legend={legend}
+            onLegendSelectChanged={onLegendSelectChanged}
+            showTimeInTooltip
+            grid={{
+              left: 8,
+              right: 8,
+              top: 20,
+              bottom: 0,
+              ...releaseBubbleGrid,
+            }}
+            tooltip={{
+              formatAxisLabel: (
+                value,
+                isTimestamp,
+                utc,
+                showTimeInTooltip,
+                addSecondsToTimeFormat,
+                _bucketSize,
+                _seriesParamsOrParam
+              ) =>
+                String(
+                  defaultFormatAxisLabel(
+                    value,
+                    isTimestamp,
+                    utc,
+                    showTimeInTooltip,
+                    addSecondsToTimeFormat,
+                    bucketSize
+                  )
+                ),
+            }}
+            yAxis={{
+              splitNumber: 2,
+              minInterval: 1,
+              axisLabel: {
+                formatter: (value: number) => {
+                  return formatAbbreviatedNumber(value);
+                },
+              },
+            }}
+            xAxis={{
+              ...releaseBubbleXAxis,
+            }}
+            {...(disableZoomNavigation
+              ? {
+                  isGroupedByDate: true,
+                  dataZoom: chartZoomProps.dataZoom,
+                }
+              : chartZoomProps)}
```

Comment:
For the chart inside of the drawer -- we dispatch a "zoom" action to it so that it sets the proper time boundaries based on the release bucket timestamps.